sdk/core/azure-core/azure/core/pipeline/base.py

```diff
@@ -80,7 +80,7 @@ class Pipeline(AbstractContextManager, Generic[HTTPRequestType, HTTPResponseType
     """
 
     def __init__(self, transport, policies=None):
-        # type: (HttpTransport, List[Union[HTTPPolicy, SansIOHTTPPolicy]]) -> None
+        # type: (Any, List[Union[HTTPPolicy, SansIOHTTPPolicy]]) -> None
```

Comment:
1. I don't think HttpTransport is any
2. Shouldn't List[Union[HTTPPolicy, SansIOHTTPPolicy] be PoliciesType?

Reply:
included this